### PR TITLE
User Profile Page - Static Screen

### PR DESCRIPTION
**Pull request by SukanyaSen530**
- Post Card
- Image Card
- Profile Page

### CODE REVIEW

File: src/components/Cards/post-card.scss [hunk 1]
```diff
@@ -2,5 +2,5 @@
   width: 100%;
   box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 2px 6px 2px;
   margin: 2rem 0;
-  background: whitesmoke;
+  background: white;
```
Comment (ArjunGTX) [inline src/components/Cards/post-card.scss:5]: better to store colors in variables

**Review by KanekarSnehal — APPROVED** [review]
Clean code
No issues found, good to merge

File: src/components/Sidebar/sidebarData.js [hunk 2]
```diff
@@ -1,22 +1,35 @@
...
+    tooltip: "people",
+    path: "/home/people",
+    name: "people",
+  },
+  {
+    id: "m4",
     icon: <FaUserAlt />,
     tooltip: "profile",
-    path: "/home/profile/:userId",
+    path: "/home/profile/",
```
Comment (ArjunGTX) [inline src/components/Sidebar/sidebarData.js:32]: you can delcare routes as constants

**Review by ArjunGTX — APPROVED** [review]
Looks good to merge

File: src/components/Sidebar/sidebarData.js (hunk 2 above)
Comment (SukanyaSen530) [reply src/components/Sidebar/sidebarData.js:32]: Sure, will do that.

File: src/components/Cards/post-card.scss (hunk 1 above)
Comment (SukanyaSen530) [reply src/components/Cards/post-card.scss:5]: yes, will do that last :)

File: src/pages/Profile/UserPosts.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import { post } from "../../dummy";
+
+import "./common-user.scss";
+
+function UserPosts() {
+  return (
+    <div className="image-container">
+      {Array(5)
+        .fill(0)
```
Comment (sharathnair9999) [inline src/pages/Profile/UserPosts.jsx:12]: Good practice to test this way👍

**Review by sharathnair9999 — APPROVED** [review]
Code is clean good work

**Pull request merged**